### PR TITLE
Mv work

### PR DESCRIPTION
**Pull request by ch4ne5teban**
In this pull request, I have created sections for showcasing my projects. Thank you in advance for your feedback.

### CODE REVIEW

**Review by gtekle — CHANGES_REQUESTED** [review]
Hi @ch4ne5teban :wave: , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that Stylelint linter failing. Kindly fix it.
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -79,6 +79,82 @@ <h2 class="profession">Software Developer</h2>
       <img src="./assets/lists/downarrow.png" alt="arrow pointing downward" />
       <span class="work">Work</span>
     </div>
+    <h2 class="recent-work">MY RECENT WORK</h2>
+    <section id="portfolio">
+      <div class="project-container">
+        <div class="project-imgholder"></div>
```
Comment (gtekle) [inline index.html:85]: - [ ] Kindly the alignment of the project image as in the Figma design. I see that you used a `div` on behalf of an image. However, this is going to hold snapshot image of a project. So, what do you think if you can use a sample image for now and later in the module you will have to add different projects with different snapshot images to you profile?
#### Figma
image
#### Impl
image

**Review by gtekle — CHANGES_REQUESTED** [review]
Hi @ch4ne5teban :wave: , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
this-close


## Highlights
:heavy_check_mark: Clean code
:heavy_check_mark: Nice UI
:heavy_check_mark: Good use of Github flow
:heavy_check_mark: Descriptive PR title and summary
:heavy_check_mark: Meaningful commit messages

## Required Changes ♻️
_Check the comments under the review._

- [x] You should left align the list of Languages and Frameworks section as in the Figma design.
     #### Design      
image
     #### Impl
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ch4ne5teban) [reply index.html:85]: @gtekle Thank you for your feedback. Do you have any suggestions on how I could achieve the left alignment of the lists? I tried using different combinations of flexbox properties on the container and the items but nothing seems to help.

Comment (gtekle) [reply index.html:85]: @ch4ne5teban 
Did you try using `justify-content: space-start;` in the #lists css selector on behalf of `justify-content: space-around;'?

Comment (ch4ne5teban) [reply index.html:85]: I just did! Thanks, I tried flex-start.

**Review by bimbolabuari — APPROVED** [review]
Hi @ch4ne5teban, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**